Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs

```diff
@@ -251,6 +251,91 @@ private void UpdateIcon()
 
         private void UpdateButtons()
         {
+#if NET6_0
```

Comment:
This scope is "Way too big"
Just needed around the single case line statement, 
With `#else` binging the actual int number

Reply:
Yes, just thought about that... will "fix" now

Reply:
Fixed